tests/connection/connection.cfg

```diff
@@ -1,4 +0,0 @@
-[imc]
```

Comment:
Its the adding to the .gitignore that also deletes it as part of the commit.  I'm not sure we can do one without the other?